src/extension.ts

```diff
@@ -242,7 +242,10 @@ function activateCheckFile(context: vscode.ExtensionContext) {
 
         // Only check rego files
         if (doc.languageId === 'rego') {
-            let args: string[] = ['check'];
+            let args: string[] = ['check']; 
+            if (opa.canUseStrictFlag()) {
```

Comment:
[nit] "can" is a little misleading, I think. We _can_ if we have the proper version; but we only _do_ when it's also configured to do so, don't we?

Reply:
Yea, I think I agree. I was going with an already established pattern. What do you think about `shouldUseStrictFlag()` or merely `useStrictFlag()`?